test/agentchat/test_groupchat.py

```diff
@@ -504,6 +490,103 @@ def test_selection_helpers():
         groupchat.manual_select_speaker()
 
 
+def test_init_default_parameters():
+    agents = [Agent(name=f"Agent{i}") for i in range(3)]
+    group_chat = GroupChat(agents=agents, messages=[], max_round=3)
+    for agent in agents:
+        assert set([a.name for a in group_chat.allowed_speaker_transitions_dict[agent]]) == set(
+            [a.name for a in agents]
+        )
+
+
+def test_graph_parameters():
+    agents = [Agent(name=f"Agent{i}") for i in range(3)]
+    with pytest.raises(Exception):
```

Comment:
Can we use more specific exception here? Same for line 513.

Reply:
Done